src/components/editor/app-bar/help-button/help-button.tsx

```diff
@@ -6,7 +6,7 @@ import { Cheatsheet } from './cheatsheet'
 import { Links } from './links'
 import { Shortcut } from './shortcuts'
 
-enum HelpTabStatus {
+export enum HelpTabStatus {
```

Comment:
Why does this need to be exported?

Reply:
If I don't do that, then the linter shows a warning, that HelpTabStatus is unused.

Reply:
Ah okay.